bin/bench

```diff
@@ -45,6 +47,13 @@ with open(sbt_bench_log_path, "r") as sbt_bench_log:
     if m:
       row = m.groups()
       for j, cell in enumerate(row): lengths[j] = max(len(cell), lengths[j])
+      ops = float(re.search("(\d+.\d+)", row[3]).groups()[0])
+      if not first:
+        firstOps = ops
+        first = True
+      else:
+        overhead = "(+ {}%)".format(int(math.ceil((ops / firstOps - 1) * 100)))
+        row = row + (overhead, )
```

Comment:
How does the updated `bench/README.md` file look like with this change?

Reply:
```
Benchmark                      Mode    Cnt Score
QuickScalacBaseline            sample  50  1042.956 ± 17.837 ms/op   
QuickScalacRangepos            sample  50  1089.995 ± 11.297 ms/op   (+5%)
QuickScalametaBaseline         sample  40  1294.257 ± 22.686 ms/op   (+25%)
QuickScalametaFullContents     sample  40  1295.044 ± 11.326 ms/op   (+25%)
QuickScalametaFullDenotations  sample  40  1395.760 ± 12.056 ms/op   (+34%)
QuickScalametaFullOverrides    sample  38  1427.002 ± 21.510 ms/op   (+37%)
QuickScalametaFullSynthetics   sample  40  1345.795 ± 15.959 ms/op   (+30%)
```